### PR TITLE
Patrick day2

### PR DESCRIPTION
**Pull request by pwkurth**

### CODE REVIEW

File: pwkurth/node/day_2/problem2_part2.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+function horizontalPosition(input) {
+
+    let total_aim = 0
+    let total_depth = 0
+    let horizontal_position = 0
+    let inital_aim = 1
```
Comment (RookieOne) [inline pwkurth/node/day_2/problem2_part2.js]: initial is misspelled :)

File: pwkurth/node/day_2/problem2_part2.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            //Our inital aim is set to 1, because their is no inital aim (0)
+            if (inital_aim === 1) {
+                //So we take our position and times it by 1 forward 5 adds 5 but no aim
+                total_depth = horizontal_position * 1
+            }
+            //If our initial_aim isn't 1 (which will be the case after the first if)
+            //Take our horizontal_position * total_aim
+            //Problem, we only want that first number, NOT the total (horizonotal_position)
+            //So create new variable, that adds the total_depth
+            else if (inital_aim != 1) {
```
Comment (RookieOne) [inline pwkurth/node/day_2/problem2_part2.js]: !==

File: pwkurth/node/day_2/readPuzzle.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            if (i % 2) {
+                //push the array, and make number as int
+                arr.push(+s);
+                //otherwise we expect it to be a string
+            } else {
+                arr.push(s);
+            }
+        }
+    }
+    return arr;
```
Comment (RookieOne) [inline pwkurth/node/day_2/readPuzzle.js:40]: Add comments to explain input and output. With examples

```
input:

"forward 5"

output:

[
  "forward ",
  5
]
```

**Review by RookieOne — APPROVED** [review]

File: pwkurth/node/day_2/readPuzzle.js (hunk 3 above)
Comment (pwkurth) [reply pwkurth/node/day_2/readPuzzle.js:40]: Added a sample input/output

**Pull request merged**